spec/controllers/admin_controller_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+describe AdminController do
+  describe '#index' do
+    it 'should be successful' do
+      get :index
+      expect(response).to be_successful
```

Comment:
You aren't setting any user here, so you're testing that an anonymous user is going to get to the index action successfully (which they won't because we're calling `authorize!` in the index action).  This should be a test, but we should be asserting that CanCan is throwing an access denied error for anonymous users. (see other places where we test `CanCan::AccessDenied` since asserting errors thrown is a little tricky).

You can also have another test (or multiple for site admins, super admins, normal webauth users) where you stub the `current_user` w/ a created user of that type.

```ruby
stub_current_user(create(:superadmin_user))
stub_current_user(create(:site_admin_user))
stub_current_user(create(:webauth_user))
```